tests/StacktraceTest.php

```diff
@@ -241,6 +241,45 @@ public function buildFromBacktraceDataProvider(): \Generator
                 ],
             ],
         ];
+
+        yield 'Backtrace with frame containing memory address $ format' => [
```

Comment:
What about changing the description to something more meaningful, e.g. `Backtrace with frame containing memory address in PHP >=7.4.2 format`?